search/processor.cpp

```diff
@@ -644,22 +633,15 @@ void Processor::InitParams(QueryParams & params)
       if (type >= 0)
       {
         ASSERT(type == 70 || type > 4000, (type));
```

Comment:
As you can see, this code is called only for old mwms, where was a different numeration for types. I can't explain you what do these constants mean in the context of type indexing, this is a legacy code.